.github/ISSUE_TEMPLATE/add-development-request.md

```diff
@@ -8,3 +8,7 @@ assignees: ''
 ---
 
 
+Please ckeck [Quick Start](https://github.com/NOAA-EMC/WW3/wiki/Quick-Start), [FAQ page](https://github.com/NOAA-EMC/WW3/wiki/FAQs-page) and [GitHub Discussions](https://github.com/NOAA-EMC/WW3/discussions) pages before proceeding with the issue[^1].     
```

Comment:
Can we Preview this somewhere? Maybe I'm not understanding correctly, but it'd be nice to add a category of user questions that then gives this information and direct someone to discussions page instead or some sort of top level statement instead of adding into each of the other issue types.  Isn't this the text that would pop up if you create an issue?

Reply:
@JessicaMeixner-NOAA You can preview them here https://github.com/aliabdolali/WW3/blob/fb_update_issue_temp/.github/ISSUE_TEMPLATE/add-development-request.md

At first, I wanted to add them somewhere under isssue section (top of it), not inside templates. But from my search, there is no such a way. That is why I decided to add them to each template, so the users read it and do not make issues if it is not an issue.

Reply:
Okay, I still think maybe just another issue type instead of adding this statement to all the others would be my preferred solution.  Let's see what Chris thinks.